md_eval_function.m

```diff
@@ -16,7 +16,12 @@
             md_funcs{s}{d},params,t);
     end
     
+    % Captain! This time multiplier can just be factored out. You factor out time multiplier
```

Comment:
possibly, but I'm not sure this is true in the general case

Reply:
Asgard is designed as if that statement is true, so I just followed the pattern.